storage-s3-dynamodb/src/main/java/io/delta/storage/S3DynamoDBLogStore.java

```diff
@@ -267,22 +267,34 @@ private void tryEnsureTableExists(Configuration hadoopConf) throws IOException {
                 LOG.info("Waiting for `{}` table creation", tableName);
                 try {
                     Thread.sleep(1000);
-                } catch(InterruptedException e) {
+                } catch (InterruptedException e) {
                     throw new InterruptedIOException(e.getMessage());
                 }
             } else {
                 LOG.error("table `{}` status: {}", tableName, status);
                 break;  // TODO - raise exception?
             }
-        };
+        }
+        ;
+    }
+
+    private boolean supportsHadoopFSCCredentials(Class<?> awsCredentialsProvider) {
```

Comment:
what is `FSC` in this? 
Also, how about `readsCredsFromHadoopConf`? more intuitive that "support"?

```suggestion
    private boolean supportsHadoopFSCCredentials(Class<?> awsCredentialsProviderClass) {
```
to differentiate class vs instance of the class

Reply:
agree.